src/main/scala/com/mozilla/telemetry/pings/package.scala

```diff
@@ -0,0 +1,225 @@
+package com.mozilla.telemetry
+
+import java.sql.Timestamp
+
+import com.mozilla.telemetry.heka.Message
+import org.json4s._
+import org.json4s.jackson.JsonMethods.parse
+
+package object pings {
```

Comment:
While I like the principle, I think this would be a pain to handle in practice for 2 reasons:
1- Changes to a central ping declaration (think of a new non-optional field) will require changes in every place where that ping is used, across different repositories. 
2- Different projects may need different parts of a ping. For example in this patch I kept the definition of the pings intentionally loose. This made it easier to create test fixtures and it likely keeps the memory footprint at a minimum (not too sure about this though).
That said, this is the second time that I spend time defining the structure of a crash ping as a case class, I can see the benefits of moving this to moztelemetry.
@vitillo @fbertsch final thoughts?